### PR TITLE
VA forms: Add form 21P-0847

### PR DESCRIPTION
**Pull request by gmrabian**
Add form to va forms_api capabilities

## Summary

- Add form id to controller-accepted values
- Add generated model and mapping files, with updated values to match frontend
- Add custom stamp method for signature field
- Add test payloads with minimum and maximum fields
- Enable tests for form
- Add PDF for form generation

## Related issue(s)
- https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/309
- 

## Testing done

- Tested locally with tests and with frontend

## What areas of the site does it impact?
21P-0847 (non prod)

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: modules/forms_api/app/services/forms_api/pdf_stamper.rb [hunk 1]
```diff
@@ -69,6 +69,27 @@ def self.stamp2110210(generated_form_path, data)
...
+
+      stamp_path = Common::FileHelpers.random_file_path
+      Prawn::Document.generate(stamp_path, margin: [0, 0]) do |pdf|
+        pdf.start_new_page
+        pdf.draw_text signature_text, at: desired_stamps[0], size: 16
+      end
+
+      multistamp(generated_form_path, stamp_path)
+    rescue
+      Rails.logger.error "Failed to generate stamped file: #{e.message}"
```
Comment (jeremy6d) [inline modules/forms_api/app/services/forms_api/pdf_stamper.rb:87]: I don't think this is going to work unless you make it `rescue => e`. And I'd really recommend capturing specific exceptions rather than any exception, but I won't block you on that.

Comment (jeremy6d) [reply modules/forms_api/app/services/forms_api/pdf_stamper.rb:87]: and maybe test exception handling if you're going to leave it so open ended?

**Review by jeremy6d — CHANGES_REQUESTED** [review]
See comments, should be small change, and the pipeline looks messed up anyway re: that one failing code check

File: modules/forms_api/app/services/forms_api/pdf_stamper.rb (hunk 1 above)
Comment (gmrabian) [reply modules/forms_api/app/services/forms_api/pdf_stamper.rb:87]: Thanks for calling that out! I refactored the stampers to cut down on that reused logic, added the ` => e` and added tests for errors.

Comment (gmrabian) [reply modules/forms_api/app/services/forms_api/pdf_stamper.rb:87]: I still left the error generic because there are a few failure points, and which one it is isn't important for us right now. I will keep that suggestion in mind.

Comment (jeremy6d) [reply modules/forms_api/app/services/forms_api/pdf_stamper.rb:87]: It's totally your call

**Review by jeremy6d — APPROVED** [review]
